browser-test/src/civiform_admin_question_list.test.ts

```diff
@@ -134,19 +135,23 @@ describe('Admin question list', () => {
 
     await adminQuestions.gotoAdminQuestionsPage()
 
-    await page.locator('#question-bank-sort').selectOption('adminname-asc')
+    await adminQuestions.sortQuestions('adminname-asc')
     expect(await adminQuestions.questionBankNames()).toEqual(['a', 'b', 'c'])
-    await page.locator('#question-bank-sort').selectOption('adminname-desc')
+    await validateScreenshot(page, 'questions-list-sort-dropdown-last-adminname-asc')
+    await adminQuestions.sortQuestions('adminname-desc')
     expect(await adminQuestions.questionBankNames()).toEqual(['c', 'b', 'a'])
-    await page.locator('#question-bank-sort').selectOption('numprograms-asc')
+
+    // Question 'b' is referenced by 0 programs, 'c' by 1 program, and 'a' by 2 programs.
+    await adminQuestions.sortQuestions('numprograms-asc')
     expect(await adminQuestions.questionBankNames()).toEqual(['b', 'c', 'a'])
-    await page.locator('#question-bank-sort').selectOption('numprograms-desc')
+    await adminQuestions.sortQuestions('numprograms-desc')
     expect(await adminQuestions.questionBankNames()).toEqual(['a', 'c', 'b'])
-    await page.locator('#question-bank-sort').selectOption('lastmodified-desc')
+
+    // Question 'c' was modified after question 'a' which was modified after 'b'.
```

Comment:
Weren't a and c published at the same time, so they should have the same lastModified? Meaning the order is arbitrary

Reply:
Publishing doesn't count as a modification.

Reply:
Got it